Enterprise-CMCS/managed-care-review
# MCR-4123 Fix contract actions missing on certain submissions

## Summary
Resolves an issue where a specific VAL submission (see ticket) is missing some of its related contract actions 

#### Related issues
https://jiraent.cms.gov/browse/MCR-4123

#### Test cases covered
- I tested this by pulling down VAL data locally. I saw that the faulty rate submission summary for  **MCR-FL-0210-PCCME-RATE-20240502-20240509-CERTIFICATION-20240404** was loading correctly.
- Tried for awhile to write a failing test. I wasn't able to achieve that. The test continues to pass with or without the fix added.
   - I have added the test as `skip` for now.
   - I would like a reviewer to take a look at the test. Do you see an option for what I should do to try and get a legitimate failing test for the bug that resolves for the fix?  Otherwise, I will remove the test, just merge this fix as is without a test.


## QA guidance
- QA regression check for Contract Actions behavior on rate summary - add, remove, edit linked rates and check that rate summary still displays as expected
